spec/view_objects/allocations_view_spec.rb

```diff
@@ -132,4 +132,187 @@
       it { is_expected.to be_empty }
     end
   end
+
+  describe "when allocation period is closed" do
```

Comment:
```suggestion
  context "when allocation period is closed" do
```

`context` is for state right?

https://github.com/DFE-Digital/publish-teacher-training/blob/7661aa80a0164e0ac4d298d5bae3ea0b63c7aff1/spec/models/allocation_spec.rb#L12

Reply:
I think it reads better "describe when ...."

Reply:
Ok, i'll change these